### PR TITLE
Fix/Signing in go `v1.19`

### PR DESCRIPTION
**Pull request by carpawell**

### CODE REVIEW

File: crypto/ecdsa/signer.go [hunk 1]
```diff
@@ -31,7 +31,16 @@ func (x Signer) Sign(data []byte) ([]byte, error) {
...
+	params := elliptic.P256().Params()
+	curveOrderByteSize := params.P.BitLen() / 8
+
+	buf := make([]byte, 1+curveOrderByteSize*2)
+	buf[0] = 4
+
+	_ = r.FillBytes(buf[1 : 1+curveOrderByteSize])
+	_ = s.FillBytes(buf[1+curveOrderByteSize:])
+
+	return buf, nil
```
Comment (KirillovDenis) [inline crypto/ecdsa/signer.go:43]: Shouldn't we move this to a separate function and add the same comments as in [neofs-crypto](https://github.com/nspcc-dev/neofs-crypto/blob/d29d7f9c3e8d6b6c14161af210e8aa1fa85bf36f/ecdsa.go#L42-L54)

**Review by KirillovDenis — APPROVED** [review]
Checked this PR in neofs-http-gw. It doesn't panic now

**Review by fyrchik — APPROVED** [review]

File: crypto/ecdsa/signer.go (hunk 1 above)
Comment (carpawell) [reply crypto/ecdsa/signer.go:43]: > move this to a separate function

that is used exactly in one place with the known points so i guess we do not need a separate function for that

> add the same comments

what exact comments do you mean?

**Comment by KirillovDenis** [comment]
Shouldn't this PR close https://github.com/nspcc-dev/neofs-sdk-go/issues/316?

**Pull request merged**